sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/generator/AbstractQueryGenerator.java

```diff
@@ -2,25 +2,27 @@
 // Licensed under the MIT License.
 package com.azure.spring.data.cosmos.core.generator;
 
-import com.azure.cosmos.models.SqlParameter;
-import com.azure.cosmos.models.SqlQuerySpec;
-import com.azure.spring.data.cosmos.core.query.Criteria;
-import com.azure.spring.data.cosmos.core.query.CriteriaType;
-import com.azure.spring.data.cosmos.core.query.DocumentQuery;
-import com.azure.spring.data.cosmos.exception.IllegalQueryException;
+import static com.azure.spring.data.cosmos.core.convert.MappingCosmosConverter.toCosmosDbValue;
```

Comment:
We add static imports at the end. We also sort the imports alphabetically and by groups. I have sent you the intellij code style file to import in your intellij to fix these imports. Please use that to optimize these ..